.circleci/config.yml

```diff
@@ -53,7 +74,7 @@ workflows:
 
       - run-task:
           name: backup environment
-          requires: [ api unit test, client unit test, integration test ]
+          requires: [ integration test ]
           filters: { branches: { ignore: [ main ] } }
           task_name: backup
```

Comment:
Probably not part of this PR but I think we should move the backup and restore steps to the weekly test run we do